sdk/nodejs/__tests__/initialize.spec.ts

```diff
@@ -0,0 +1,36 @@
+import { DVCClient, DVCCloudClient, initialize } from '../src/index'
+
+jest.mock('../src/bucketing')
+jest.mock('../src/environmentConfigManager')
+
+describe('NodeJS SDK Initialize', () => {
+    afterAll(() => {
+        jest.clearAllMocks()
+    })
+
+    it('sucessfully calls initialize with no options', async () => {
+        const client: DVCClient = await initialize('token').onClientInitialized()
+        expect(client).toBeDefined()
+    })
+
+    it('fails to initialize in Local Bucketing mode when no token is provided', async () => {
+        try {
+            await initialize(undefined as unknown as string)
+            // Fail test if above expression doesn't throw anything.
```

Comment:
you can also write this as :
`await expect(() => initialize(undefined as unknown as string)).rejects.toThrow('Missing environment key! Call initialize with a valid environment key')`

Reply:
Ah! Thanks.. that's what I couldn't figure out how to write correctly.. that's why I went with the try/catch format.. will revise it! thanks 🙏